shared/checks/oval/installed_env_has_yum_or_zypper_package.xml

```diff
@@ -0,0 +1,38 @@
+<def-group>
+  <definition class="inventory"
+  id="installed_env_has_yum_or_zypper_package" version="1">
+    <metadata>
+      <title>Package yum or zypper is installed</title>
+      <affected family="unix">
+        <platform>multi_platform_all</platform>
+      </affected>
+      <description>Checks if package yum or zypper is installed.</description>
+      <reference ref_id="cpe:/a:yum_or_zypper" source="CPE" />
+    </metadata>
+    <criteria>
+      <extend_definition comment="Package yum is installed" definition_ref="test_env_has_yum_installed" />
```

Comment:
Reference the definition, `installed_env_has_yum_package`.
This should fix the build issue.